### PR TITLE
Pass boskos credentials to ovirt jobs [example PR, do not merge]

### PR DESCRIPTION
**Pull request by petr-muller**

### CODE REVIEW

File: ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml [hunk 1]
```diff
@@ -121,32 +130,27 @@ objects:
 
         trap 'rc=$?; CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi; if test "${rc}" -ne 0; then touch /tmp/shared/exit; fi; exit "${rc}"' EXIT
 
-        # hack for bazel
```
Comment (stevekuznetsov) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml:124]: I think they would still need this bit, though.

Comment (petr-muller) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml:124]: Not really, I took this from https://github.com/openshift/release/pull/11520

Comment (petr-muller) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml:124]: Only to make the template work, that PR was not merged when I started :)

**Comment by petr-muller** [comment]
/close

Was opened only as an example, close

**Pull request closed without merging**